koku/koku/test_rbac.py

```diff
@@ -20,6 +20,28 @@
 from koku.rbac import RbacConnectionError
 from koku.rbac import RbacService
 
+
+def create_default_access(access_dict={}, default_write={"write": [], "read": []}, default_read={"read": []}):
+    """Helper function for defaulting access permissions."""
+    default = {
+        "cost_model": default_write,
+        "settings": default_write,
+        "aws.account": default_read,
+        "aws.organizational_unit": default_read,
+        "azure.subscription_guid": default_read,
+        "gcp.account": default_read,
+        "gcp.project": default_read,
+        "openshift.cluster": default_read,
+        "openshift.node": default_read,
+        "openshift.project": default_read,
+        "ibm.account": default_read,
+        "oci.payer_tenant_id": default_read,
+    }
+    for key, value in access_dict.items():
+        default[key] = value
+    return default
```

Comment:
You can use the `update()` method to modify the values in `default` or the `|` operator to return a new dictionary with merged values. 
```suggestion
    return default | access_dict
```